OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please unstage this.

Reply:
Weird that this still appears. Maybe it goes away when you squash the commits together.

Reply:
@abcdefg30 would you care to verify the squashed commits? :)

Reply:
The squashing worked, yes. It'd be nice if you could fix the commit message, however. (Just use the message of the first commit but nothing more). Unfortunately, the change here didn't go away, so please try unstaging/removing it again. (Don't hesitate to ask if you need help with this.)

Reply:
Yeah I can use some help with this specific unstage....:)

Reply:
I don't know which git client you use, so here is the git-bash version.
```
git reset --soft head~1
git reset OpenRA.Game/OpenRA.Game.csproj
git checkout OpenRA.Game/OpenRA.Game.csproj
```
The first command deletes your commit and moves the files back into the staging area.
The second command moves this file out of the staging area, so the third command can remove the change.
After that is done, just re-commit your changes and push again.
Hope that helps. :)

Reply:
Hey @abcdefg30 . how is it looking? had more trouble with git than the actual fix... are we good now?

Reply:
Sorry, something must have gone wrong, the change is still there. To make things worse, some unwanted commits sneaked in which means you will have to rebase again.
Yes, git can be hard to work with... You might want to join our [IRC channel](https://webchat.freenode.net/?channels=openra) for direct help. (Unfortunately, the channel is restricted to registered accounts currently...)
Worst case one of us (probably me ^^) could do the nasty git work for you, if you wish. (I assume you have "Allow edits from maintainers" enabled for this PR.)

Reply:
@abcdefg30 sure go ahead.

Reply:
Done.